cmd/test_load.go

```diff
@@ -0,0 +1,199 @@
+package cmd
```

Comment:
While reading this code I noticed there are a lot of Homonyms, which made it quite confusing for me. The main helper function is called `loadTest`, which I intuitively expected to: perform a load test. 

My understanding is that what it does is actually load the configuration, state, and perform setups that are prerequisites to perform a test. Thus, **I would suggest a different verb than *load* in order to avoid confusion?**
Maybe `prepare`, or `configure`, or `assemble`: `prepareTest`, `configureTest`, or `assembleTest`? We could even prepend those verbs to the existing name: `prepareLoadTest`, `configureLoadTest`, or `assembleLoadTest`? what do you think?

**The term `test` is also equivoqual**, because it's so polysemic. Personally, when I read it, I immediately think of a unit/integration test, but used in the context of a load testing software, it also binds to the idea of running a load test. In this specific context, the best idea I can come up with, paired with the idea of dropping the term `load` to avoid homonymy would be to use something like `run` or `testrun`, or `loadtestrun` instead (something along those lines, to explicit its nature). What do you think?

Another suggestion I have is to use this opportunity to document every symbols we introduce here properly. Even if they are private, I think we would benefit from having doc for most of them (at least types and functions). That would make our future selves and future team members lives easy. Even just two sentences to explain what the function/symbol does, and how/where it's meant to be used (its reason to be).

Reply:
:thinking: you raise some good points, but as always, naming things well is going to be hard... :sweat_smile: 

`load` can indeed be confusing... :disappointed: Even though, AFAIK, we don't use any variations of the term "load testing" or "load test" internally anywhere in the k6 codebase, we use it in our marketing website and documentation, so contributors could be confused.

On the other hand, `configure`, `assemble`, etc. don't reflect the reality of what is happening under the hood of that function, so they can be confusing in that way. So my suggestion is to rename this function to `loadAndConfigureTest()`, which both accurately reflects what's going on, and doesn't allow any conflation with "load testing", it's very clearly about "loading a test". WDYT?

Regarding the term "test", I think it's unavoidable and already pretty well defined in the context of k6. k6 is a tool that runs tests. And execution `k6 run test.js` is a "testrun". And with future support for test suites (https://github.com/grafana/k6/issues/1342, something that was one of the goal of my recent PRs to unlock), k6 will be able to run multiple tests with a single `k6 run`, e.g. `k6 run test1.js test2.js`. So the term will be even more clearly defined. 

While there might be a small confusion for a new developer between a k6 "test" and a Go test of k6 itself that lives in a `_test.go` file, that seems like the lesser evil and something that will be clear after the first few minutes of looking at the code.

Reply:
All in on `loadAndConfigureTest` 👍🏻 
Fair enough about the usage of "test" 👍🏻 🙇🏻

Reply:
Resolved by https://github.com/grafana/k6/pull/2412/commits/c0bc707e030326ae2038db0940071b27493487f7